<!--
  Copyright (C) 2019-2021 LEIDOS.
  Licensed under the Apache License, Version 2.0 (the "License"); you may not
  use this file except in compliance with the License. You may obtain a copy of
  the License at
  http://www.apache.org/licenses/LICENSE-2.0
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  License for the specific language governing permissions and limitations under
  the License.
-->
<!--
  This file is used to launch the CARMA3 motion_computation node
-->

<launch>
  <node name="motion_computation" pkg="motion_computation" type="motion_computation_node">
    <rosparam command="load" file="$(find motion_computation)/config/parameters.yaml"/>
    <remap from="incoming_mobility_path" to="$(optenv CARMA_MSG_NS)/incoming_mobility_path"/>
  </node>
</launch>
